src/Lecture3.hs

```diff
@@ -57,7 +57,16 @@ of a weekday.
 >>> toShortString Monday
 "Mon"
 -}
-toShortString = error "TODO"
+toShortString :: Weekday -> String  
+toShortString Monday = "Mon"
+toShortString Tuesday = "Tue"
+toShortString Wednesday = "Wed"
+toShortString Thursday = "Thu"
+toShortString Friday = "Fri"
+toShortString Saturday = "Sat"
+toShortString Sunday = "Sun"
+-- not sure if this was the intent of this exercise?
```

Comment:
You're correct, this was not the intent of this exercise 🙂 
You can use standard functions `show` to implement this one shorter 🩳